pex/interpreter.py

```diff
@@ -124,7 +124,7 @@ def get(cls, binary=None):
         # Pex identifies interpreters using a bit of Pex code injected via an extraction of that
         # code under the `PEX_ROOT` adjoined to `sys.path` via `PYTHONPATH`. We ignore such adjoined
         # `sys.path` entries to discover the true base interpreter `sys.path`.
-        pythonpath = frozenset(os.environ.get("PYTHONPATH", "").split(os.pathsep))
+        pythonpath = frozenset(os.environ.get("PYTHONPATH", "").split(":"))
```

Comment:
Wholesale fixed these. Python defines the correct character under all OSes as ":".